conftest.py

```diff
@@ -9,6 +11,18 @@ def pytest_addoption(parser):
         "--package_name", action="append", default=[],
         help="list of package_name to pass to test functions",
     )
+    parser.addoption(
+        "--integration", action="store_true", default=False,
+        help="run integration tests (only)"
+    )
+
+
+def pytest_configure(config):
+    config.addinivalue_line("markers", "integration: indicate integration tests")
+
+    if config.option.integration:
+        # Assume unit tests and flake already run
+        config.option.flake8 = False
```

Comment:
Running `flake` again during integration would take more time.

Reply:
Instead, you can pass `-p no:flake8` to the pytest run.

Reply:
Actually, rather than install the plugin and then disable it, you can instead provide separate sets of extras for the integration tests and just not include the unwanted plugins when doing integration tests.

Reply:
Thanks @jaraco. I added a ``testing-integration`` to extras_require.

There is some small amount of duplication between ``testing-integration`` and ``testing`` but hopefully it would not add a lot of double bookkeeping.